wpeverest/everest-forms
# Dev - Airtable Compatibility

### All Submissions:






### Changes proposed in this Pull Request:



This, PR add AirTable integration feature.

### How to test the changes in this Pull Request:

1. Go to Everest Forms >> Global Settings >> Integrations >> Airtable.
2. Get Access token and connect Api.
3. Go to Everest Forms >> Form Builder Settings >> Integration >> Airtable
4.  Select Base and Schema and map fields.
5. Check Airtable whether data are sync or not.
Note : It contains PR on Free and Pro.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Airtable Compatibility
